### DISCUSSION TITLE
tauri-v1.6.1 update to tauri-v2.0.0-beta.11 migrated.json causes cargo packaging to fail

### DISCUSSION THREAD

**Discussion by nongyehong** [tauri-apps/tauri#9347] [Q&A] [answered]
error: failed to run custom berror: failed to run custom build command for `hula v1.4.0 (D:\xxx\xxx\HuLa-IM-Tauri\src-tauri)`

```shell
[✔] Environment
    - OS: Windows 10.0.26080 X64
    ✔ WebView2: 123.0.2420.65
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.77.1 (7cf61ebde 2024-03-27)
    ✔ cargo: 1.77.1 (e52e36006 2024-03-26)
    ✔ rustup: 1.27.0 (bbb9276d2 2024-03-08)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 20.10.0
    - pnpm: 8.15.4
    - yarn: 1.22.21
    - npm: 10.2.3

[-] Packages
    - tauri [RUST]: 2.0.0-beta.13
    - tauri-build [RUST]: 2.0.0-beta.10
    - wry [RUST]: 0.37.0
    - tao [RUST]: 0.26.1
    - @tauri-apps/api [NPM]: 2.0.0-beta.6
    - @tauri-apps/cli [NPM]: 2.0.0-beta.11

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:6130/
    - framework: Vue.js
    - bundler: Rollup

```

**Comment by FabianLars** [comment]
Can you please post the _full_ error log?

**Reply by nongyehong** [reply]
The console is the report error: failed to run custom berror: failed to run custom build command for hula v1.4.0 (D:\xxx\xxx\HuLa-IM-Tauri\src-tauri),I refer to upgrade documents use pnpm tauri migrate to upgrade, the src-tauri folder will generate a "capabilities/migrated json json file, the file is my v1 version allowlist configuration, When the migrated.json file exists, the cargo package will give an error: error: failed to run custom berror: failed to run custom build command for HuLa v1.4.0 (D:\xxx\xxx\ hula-im-tauri \src-tauri), but when I delete this json file, cargo can be packaged normally, But starting the project will tell me that I don't have the corresponding permissions enabled, such as window drag, getCurrent().minimize() and all these errors

**Reply by nongyehong** [reply]
![image](https://github.com/tauri-apps/tauri/assets/87641407/f872bf19-1abe-40f9-b4ba-1422cbd33100)

**Comment by nongyehong** [comment, marked as answer]
This problem should be solved by another discussion #9363
